packages/admin-ui/src/params.ts

```diff
@@ -38,7 +38,7 @@ export const wifiEnvWPA_PASSPHRASE = "WPA_PASSPHRASE";
 export const wifiEnvSSID = "SSID";
 
 // Local proxying
-export const adminUiLocalDomain = "my.dappnode.local";
+export const adminUiLocalDomain = "dappnode.local";
```

Comment:
Should this be `http://dappnode.local` and show the variable `adminUiLocalDomain` in the <a> above?

Reply:
Yes, nice catch. Done